### DISCUSSION TITLE
How to change the highlights (colors) for the plugin

### DISCUSSION THREAD

**Discussion by linkarzu** [Ramilito/kubectl.nvim#323] [Q&A] [answered]
Hey there, I've been trying to change the highlights/colors of this plugin so they match my theme colors, but I'm unable to, is something that I need to be aware of?
I change all the other highlights in neovim following this same approach, but it's not working for this plugin, does it use a different set of highlight groups or are the colors overridden when the plugin runs?
Below I show  a section on how I successfully modify any other highlights and also included the `Kubectl` ones, any help would be appreciated!

```lua
        DiagnosticInfo = { fg = colors["linkarzu_color03"] },
        DiagnosticHint = { fg = colors["linkarzu_color02"] },
        DiagnosticWarn = { fg = colors["linkarzu_color08"] },
        DiagnosticOk = { fg = colors["linkarzu_color04"] },
        DiagnosticError = { fg = colors["linkarzu_color05"] },
        RenderMarkdownQuote = { fg = colors["linkarzu_color12"] },

        -- visual mode selection
        Visual = { bg = colors["linkarzu_color16"], fg = colors["linkarzu_color10"] },

        KubectlHeader = { fg = colors["linkarzu_color04"] },
        KubectlWarning = { fg = colors["linkarzu_color03"] },
        KubectlError = { fg = colors["linkarzu_color01"] },
        KubectlInfo = { fg = colors["linkarzu_color02"] },
        KubectlDebug = { fg = colors["linkarzu_color05"] },
        KubectlSuccess = { fg = colors["linkarzu_color06"] },
        KubectlPending = { fg = colors["linkarzu_color07"] },
        KubectlDeprecated = { fg = colors["linkarzu_color08"] },
        KubectlExperimental = { fg = colors["linkarzu_color09"] },
        KubectlNote = { fg = colors["linkarzu_color10"] },
        KubectlGray = { fg = colors["linkarzu_color10"] },

```

**Comment by Ramilito** [comment, marked as answer]
Hi and thank you for bringing this to my attention!

I've had plans on doing something clever with the theme but many other features have taken prio :(
I did a quick and okay fix for now, as you mentioned, we override them when opening the plugin. Added a check for each group before assigning them in #324 so this should work now!

**Reply by linkarzu** [reply]
Niiiice, now the color overrides work, I was losing my mind and couldn't figure it out! Appreciate it!

**Reply by Ramilito** [reply]
Sorry about that, I intended to fix this way earlier (issue nr #20) and now we are at issue nr #324 😅
